### ISSUE TITLE
[BUG] misrepresented tonnage calculation in MOD->Show Details.

### ISSUE THREAD

**Issue by FJH-Photurnal** [FSGModding/FSG_Mod_Assistant#273]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When in the Show Details panel of a mod with a weight (kg icon) parameter (seen in vehicle and implement so far) the tonne conversion has a misplaced decimal. As shown 1,119 kg should convert to 1.12 t not 11.2, the lbs is correct. first noticed in version 5.1.1 (my first use).
<img width="682" alt="MisscalculationExample-FSGModAssistant" src="https://github.com/user-attachments/assets/5b4f2359-fc3a-4c25-bbd7-00aff92e345c">

### Expected Behavior

as described.

### Steps To Reproduce

right click any mod with a likely weight parameter, select Show Details and look in "Store Items".

### Environment

```markdown
- Mod Assistant Version:5.1.3
- OS: Windows 11
- Virtualized: no
```

**Comment by jtsage** [comment]
huh.  yup.  that's wrong.
![2024-11-12 21_07_01-storeItems js - FSG_Mod_Assistant (Workspace) - Visual Studio Code](https://github.com/user-attachments/assets/a3898032-8317-4c68-bb71-73f30f95e293)


will fix in next update.

**Comment by jtsage** [comment]
fixed in 5.1.4 i think (or 5.1.5).  either way, closing as complete
